ieeeuoft/newhacks-site-2024
# Adding review info for admins

## Description of change 

On the admin site, for page `admin/review/teams`, revised the fields reviewers can see when clicking on an individual team (which has 1 to 1 mapping with an individual user)

## QA approach 
- Tested whether new fields can be seen
- Tested whether existing applications prior to new fields being added still retain their existing information 
- Tested whether acceptances can still be sent

## QA questions 

1. The text fields for review/review are not spaced as well as the response on the application page. (See the 2 pictures below).

<img width="1494" alt="Screenshot 2024-10-06 at 1 22 09 PM" src="https://github.com/user-attachments/assets/19d0c139-eed6-4614-86f8-4e7bed937e33"> (On path: `http://127.0.0.1:8000/admin/review/teamreview/`)

<img width="1511" alt="Screenshot 2024-10-06 at 1 22 53 PM" src="https://github.com/user-attachments/assets/bcc1d17d-b56e-4d13-8909-6f67ddb17710"> (On path: `http://127.0.0.1:8000/admin/registration/application`)

My thought was maybe we can have logistics fetch the team name from the teams page, but they read the actual responses in `Applications` page instead of teams (since the spacing is better there). However, they would need to go back to the teams/review page to actually fill out a review. 

Another solution would be to somehow fix the spacing of the responses on the team page, but I'm not sure how to do that. 

2. I have 1 warning to give to logistics. For some reason the `Review Next Team` button isn't working (the site just hangs I think), so I advise them just not to really touch it. Was wondering whether this was an issue last year or not. 

<img width="1512" alt="Screenshot 2024-10-06 at 1 04 22 PM" src="https://github.com/user-attachments/assets/85c962c3-386a-4070-8278-d2cf46e29907">

3. In general am I missing changes anywhere else?
